demo-app/src/open-truss/components/TrinoDemo.tsx

```diff
@@ -1,16 +1,20 @@
 'use client'
 import {
+  type BaseOpenTrussComponentV1,
   BaseOpenTrussComponentV1PropsShape,
   withChildren,
```

Comment:
Thinking out loud here, what do we think about having a `BaseOpenTrussComponentV1WithChildrenPropsShape`? Children will probably be used a lot, so having this saves a couple lines (import + use). Pretty chatty name of course..

Reply:
I considered that! I thought it was too wordy hah but I see your point about number of lines... Should we drop the `BaseOpenTruss*` prefix?

- `ComponentV1`
- `ComponentWithChildrenV1Shape`